### DISCUSSION TITLE
Issues with precip optimal locations plot

### DISCUSSION THREAD

**Discussion by Adriannaluger** [earthlab-education/Earth-Analytics-2023-01-Intro#672] [! GitHub help]
Hi! I am having troubles plotting the precipitation plot to display areas with optimal vs non optimal conditions. I was successful with my soil type plot using the same code as below. However I am getting this message when trying with the precipitation layer: TypeError: '>' not supported between instances of 'Image' and 'int'. 

`# Select precipitation (cm) values from array that exceed threshold; 1=possible indiangrass, 0=not possible
precip_optimal_indiangrass = ((precip_da_clip > 28) & (precip_da_clip < 114))
precip_optimal_indiangrass.plot()
print(precip_optimal_indiangrass.shape)`

Let me know what your recommend. Thanks! @eculler

**Comment by peterkobylarz** [comment]
Hi Adri,

I had a similar issue. You need to convert the value type of the optimized raster form boolean (true/false) to integer values. You can use the following code to do this:
```python
precip_optimal_indiangrass = precip_optimal_indiangrass.astype(int)
```
Hopefully that works for you!

**Reply by Adriannaluger** [reply]
Okay, I'll try it! Thanks, Peter!!!
